.github/workflows/bindings_java_snapshot.yml

```diff
@@ -0,0 +1,130 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Deploy OpenDAL Java Snapshot
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '30 1 * * 1'
+  workflow_dispatch:
+
+jobs:
+  stage-snapshot:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            classifier: linux-x86_64
+          - os: windows-latest
+            classifier: windows-x86_64
+          - os: macos-latest
+            classifier: osx-x86_64
+          - os: macos-latest
+            classifier: osx-aarch_64
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: apache/incubator-opendal
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
```

Comment:
This one `${{ secrets.MAVEN_GPG_PRIVATE_KEY }}`.

Reply:
Added.